rocksdb_admin/admin_handler.cpp

```
@@ -415,8 +469,25 @@ AdminHandler::AdminHandler(
   CHECK(FLAGS_max_s3_sst_loading_concurrency > 0)
     << "Invalid FLAGS_max_s3_sst_loading_concurrency: "
     << FLAGS_max_s3_sst_loading_concurrency;
+
+  if (FLAGS_enable_async_delete_dbs) {
+    static const std::string db_tmp_path = FLAGS_rocksdb_dir + "db_tmp/";
+    if (!boost::filesystem::exists(db_tmp_path)) {
+      boost::system::error_code create_err;
+      boost::filesystem::create_directories(db_tmp_path, create_err);
+      if (create_err) {
+        LOG(ERROR) << "Failed to create dir: " << db_tmp_path << create_err.message();
```

Should we fail loudly in this case?